pkgs/top-level/all-packages.nix

```
@@ -40805,4 +40805,16 @@ with pkgs;
   reindeer = callPackage ../development/tools/reindeer { };
 
   charasay = callPackage ../tools/misc/charasay { };
+
+  libvoikko = callPackage ../development/libraries/libvoikko/default.nix { };
+
+  hfst = callPackage ../development/libraries/hfst/hfst.nix { };
```

Please rename hfst.nix to default.nix
```suggestion
  hfst = callPackage ../development/libraries/hfst { };
```